index.js

```diff
@@ -8,10 +8,14 @@ class Editor {
     this.server = createServer(path)
   }
 
-  open () {
-    this.server.listen(() => {
-      open(`http://localhost:${this.server.address().port}`)
-    })
+  open (silent) {
```

Comment:
```suggestion
  open ({ silent }) {
```

Flags are harder to use because `open(true)` isn't obvious, `open({ silent: true })` is